backend/app/assets/stylesheets/spree/backend/shared/_tables.scss

```diff
@@ -1,14 +1,20 @@
 table {
-  width: 100%;
-  margin-bottom: 15px;
-  border-collapse: separate;
+  // Extend all tables with Bootstrap's default table style
+  @extend .table;
 
   th, td {
-    padding: 7px 5px;
     border-right: 1px solid $color-border;
     border-bottom: 1px solid $color-border;
     vertical-align: middle;
     text-overflow: ellipsis;
+    overflow: hidden;
+
+    // Do not wrap text in all header and content cells.
+    // Overflowing content gets truncated and an ellipsis is shown.
+    // If you want to disable this, add the .wrap-text class to the column.
+    &:not(.wrap-text) {
+      white-space: nowrap;
+    }
```

Comment:
I think the default should be no wrap. Most of the time we display where little information in cells, mostly values (order numbers, email addresses, etc. You don't want to wrap these.

The only occasion where wrapping was useful (not even needed) was the image caption. It would be a much larger change set to add a no-wrapp class to all cells where we want this to happen. 

But this is just my preference. I'm also fine with having the opposite being the default. I would then vote for also making the header cells non wrapping. Nobody wants a 

"Created
at"

cell, right?